draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4240,11 +4240,17 @@ bytes=500-700,601-999
   <x:anchor-alias value="header.content-length"/>
   <iref primary="true" item="Fields" subitem="Content-Length" x:for-anchor=""/><iref primary="true" item="Header Fields" subitem="Content-Length" x:for-anchor=""/><iref primary="true" item="Content-Length header field" x:for-anchor=""/>
   <x:anchor-alias value="Content-Length"/>
-<t><cref>
-   The "Content-Length" header field indicates the number of data octets
-   (body length) for the representation. In some cases, Content-Length is
-   used to define or estimate message framing.
-</cref></t>
+<t>
+   The "Content-Length" header field indicates the associated representation's
+   data length as a decimal non-negative integer number of octets.
+   When transferring a representation in a message, Content-Length refers
```

Comment:
I am pretty sure it has that meaning regardless of version because h2 and h3 will forward it as such. This would be the case for a 206 response with one part containing a representation delimited by CL, as in

   https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#partial.single

Speaking of which, we should also define/reference that case in CL.

Reply:
HTTP has "messages" across versions, and it's a primitive used in 1.x, 2, and 3's definition of the wire format.